### DISCUSSION TITLE
Can i use this as player screen ?

### DISCUSSION THREAD

**Discussion by afkcodes** [lodev09/react-native-true-sheet#13] [General]
I was thinking to use it as an expanded player for my app, can i use it for that purpose. Also can i disable the background scale on iOS when the sheet is visible

**Comment by lodev09** [comment]
> I was thinking to use it as an expanded player for my app, can i use it for that purpose

Absolutely. I use it as an Audio Recorder. If you have issues, head over to [troubleshooting](https://sheet.lodev09.com/guides/troubleshooting) guide.

https://github.com/lodev09/react-native-true-sheet/assets/6686328/5f312b6a-aba6-4c59-b46b-ef9df6eb11c0

> Also can i disable the background scale on iOS when the sheet is visible

WIP PR #14 should let you do that. Currently working on android part.

**Reply by afkcodes** [reply]
Thanks for this, Also i tried running the example app but it fails

**Reply by lodev09** [reply]
What error are you getting?

**Reply by afkcodes** [reply]
![Screenshot from 2024-04-25 21-41-42](https://github.com/lodev09/react-native-true-sheet/assets/17390257/c290b027-2132-4993-820f-6aec2babc8e7)

**Reply by lodev09** [reply]
ah. try to use `yarn`

**Reply by afkcodes** [reply]
tried with yarn as well it didnt work

**Reply by lodev09** [reply]
I think the issue is that you're running `install` within `example` folder. Run it at the root folder :)
https://github.com/lodev09/react-native-true-sheet/blob/main/CONTRIBUTING.md

**Reply by afkcodes** [reply]
understood, thanks
